### ISSUE TITLE
Include translations from MetaMask / Brave

### ISSUE THREAD

**Issue by mapmeld** [nanexcool/defi18n#4]
I added i18n support to MetaMask, and the volunteer translations were extended and updated a lot by the Brave browser team. Their locale files contain tons of messages related to crypto in dozens of languages -- hope this covers a lot of the words and phrases which you're looking for:

https://github.com/MetaMask/metamask-extension/blob/develop/app/_locales/en/messages.json

**Comment by nanexcool** [comment]
I was looking at the Metamask translations over the weekend, and I agree they're great! Plus they have so many :D

My only concern is that for this project, I want the translations to be completely in the public domain (CC0 License) and Metamask is MIT licensed. I will talk to the MM team to see if there's anything we can do.

**Comment by mapmeld** [comment]
I agree, I hope that translations can be transferred or created in CC0. Thanks for looking into this
